clang/lib/AST/ExprConstant.cpp

```diff
@@ -14547,6 +14547,18 @@ bool FloatExprEvaluator::VisitCallExpr(const CallExpr *E) {
   default:
     return false;
 
+  case Builtin::BI__builtin_frexpf:
```

Comment:
I think it would be helpful to also update `clang\lib\AST\Interp\InterpBuiltin.cpp` to support the builtins for C++23 constexpr math functions. CC @tbaederr for his opinion.

Reply:
Not doing that is fine for now.